tests/e2e/specs/upe-split/shopper/shopper-deferred-intent-creation-upe-enabled.spec.js

```diff
@@ -134,14 +143,43 @@ describe( 'Enabled UPE with deferred intent creation', () => {
 			await expect( page ).toMatch(
 				`${ card.expires.month }/${ card.expires.year }`
 			);
+			await waitTwentySecondsSinceLastCardAdded();
```

Comment:
we need to wait before proceeding with the next test otherwise the rate limiter will be hit, so I created the separate function for this waiting logic and it's not used in this test + `afterAll`